adding_species_manuscript.tex

```diff
@@ -414,7 +418,7 @@ \section*{Additions to \stdpopsim}
 Another key feature added to \stdpopsim's simulation framework was the ability to model non-crossover recombination.
 %REVISION: clarified HGT, due to rev 3 comment
 In bacteria and archaea, recombination occurs through horizontal gene transfer, which can add new genetic material from other species (e.g., via the transfer of plasmids) or replace homologous sequences in the chromosome through homologous recombination \citep{Thomas2005,Didelot2010,Gophna2022}.
-Since this mechanism is fundamentally different from crossover recombination, such species cannot be realistically simulated with a recombination model that considers only crossovers,
+As a result, such species cannot be realistically simulated with a recombination model that considers only crossovers,
```

Comment:
removed explicit comment about the mechanism. I think the point still comes across